### DISCUSSION TITLE
`NowPlayingOverlay`'s `ProgressBar` can seek out of Track's `lastSeekablePosition`

### DISCUSSION THREAD

**Discussion by Stedoss** [ppy/osu#21441] [Q&A] [answered]
Dragging the `ProgressBar`'s value to the end of the bar results in the current track being in a strange playing state. The video below probably shows it off better than I can explain it:

https://user-images.githubusercontent.com/29103029/204159277-e0d43ac7-1040-469a-a334-5faec83752d6.mp4

It seems that the `EndTime` of the `ProgressBar` is the `CurrentTrack.Length`, however it is noted that Bass cannot seek to the end of the track, so silently fails to seek to the end of the track and ends up in this state.

**Comment by Joehuu** [comment, marked as answer]
Already tracked at https://github.com/ppy/osu/issues/9398.
